include/internal/libspdm_common_lib.h

```diff
@@ -569,18 +568,30 @@ bool libspdm_is_capabilities_flag_supported(const libspdm_context_t *spdm_contex
 bool libspdm_generate_cert_chain_hash(libspdm_context_t *spdm_context,
                                       size_t slot_id, uint8_t *hash);
 
+/**
+ * This function generates the public key hash.
+ *
+ * @param  spdm_context               A pointer to the SPDM context.
+ * @param  hash                       The buffer to store the public key hash.
+ *
+ * @retval true  public key hash is generated.
+ * @retval false public key hash is not generated.
+ **/
+bool libspdm_generate_public_key_hash(libspdm_context_t *spdm_context,
```

Comment:
The name was designed to be aligned with the function we already have for certificate chain hash generation/verification.
`bool libspdm_generate_cert_chain_hash()` <-> `bool libspdm_generate_public_key_hash()`
`bool libspdm_verify_certificate_chain_hash()` <-> `bool libspdm_verify_public_key_hash()`
Since the public key hash generation/verification process has the same routine as certificate chain does (one only **generates local hash** and **verifies peer hash**), so I tend to keep the function name as the way it is right now.